datasetinsights/datasets/coco.py

```diff
@@ -143,6 +173,90 @@ def _get_local_annotations_zip(self):
     def _get_local_images_zip(self):
         return os.path.join(self.root, f"{self.split}2017.zip")
 
+    def _get_label_mappings(self):
+        """get label mappings.
+
+        Returns:
+            dict: A dict containing {label_id: label_name} mappings.
+        """
+        ann_file_name = (
+            Path(self.root) / "annotations" / f"instances_{self.split}2017.json"
+        )
+        label_mappings = {}
+        with open(ann_file_name, "r") as ann_file:
+            anns = json.load(ann_file)
+            for cat in anns["categories"]:
+                label_mappings[cat["id"]] = cat["name"]
+        return label_mappings
+
+    @staticmethod
+    def _preprocess_dataset(data_path, split):
+        """ Preprocess dataset inside data_path and un-archive if necessary.
+
```

Comment:
Can we have unit test cases for the new added methods?

Reply:
Added.